<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultTab</class>
 <widget class="QWidget" name="ResultTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,1">
   <item row="0" column="1">
    <widget class="QWidget" name="widget_text" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_text {
	border: 1px solid lightgray;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tb_stats">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>Минимум</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>Максимум</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>Среднее</string>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>Стандартное отклонение</string>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string>Единицы измерения</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_description">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QPushButton" name="pb_export_index">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Экспорт</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QPushButton" name="pb_export_text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Экспорт</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_preview" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_preview {
	border: 1px solid lightgray;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="8,1,0">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lbl_caption">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pb_refresh">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>⟳</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="lbl_preview">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
